src/screens/OtherProfile/index.js

```diff
@@ -225,7 +325,7 @@ const OtherProfile = () => {
       );
     };
     const __renderFollowerDetail = () => {
-      if (isBlocking) return <></>;
+      if (blockStatus.blocker) return <></>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The block status of a user is now determined by `blockStatus.blocker` instead of `isBlocking`. Make sure that this new variable correctly reflects the block status.
```diff
+ if (blockStatus.blocker) return <></>;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->